runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/DataflowExecutionStateSampler.java

```diff
@@ -86,7 +86,7 @@ public void removeTracker(ExecutionStateTracker tracker) {
       return;
     }
     DataflowExecutionStateTracker dfTracker = (DataflowExecutionStateTracker) tracker;
-    completedProcessingMetrics.put(dfTracker.getWorkItemId(), dfTracker.getProcessingTimesByStep());
+    completedProcessingMetrics.put(dfTracker.getWorkItemId(), dfTracker.getProcessingTimesByStepCopy());
```

Comment:
```suggestion
    completedProcessingMetrics.put(
        dfTracker.getWorkItemId(), dfTracker.getProcessingTimesByStepCopy());
```

This should fix the spotless issues. I'm just going to apply the change since its from spotless